### DISCUSSION TITLE
Run the benchmark against  Memgraph

### DISCUSSION THREAD

**Discussion by hououou** [memgraph/memgraph#378] [Q&A] [answered]
I want to run the benchmark against Memgraph. And I notice that you may kindly provide the test [here](https://github.com/memgraph/memgraph/tree/master/tests/public_benchmark/ldbc). But unfortunately I can not find this [link](https://github.com/memgraph/memgraph/blob/ea2806bd57d8e9ef79dcb26272fc5e6838890871/tests/public_benchmark/ldbc/setup#L16). Can you give me a working link? Thanks!

**Comment by hououou** [comment]
I searched the related packages from the global website. And here the related commands:

git clone https://github.com/memgraph/ldbc-snb-impls.git
git clone https://github.com/ldbc/ldbc_driver.git
git clone https://github.com/ldbc/ldbc_snb_datagen.git
wget -nv https://dlcdn.apache.org/maven/maven-3/3.5.4/binaries/apache-maven-3.5.4-bin.tar.gz maven.tar.gz

**Reply by antejavor** [reply]
@hououou Hi there, sorry for a bit late reply, seems like our GitHub discussions sometimes go under the radar. Yeah, that [part](https://github.com/memgraph/ldbc-snb-impls.git) of the code will be deleted from the code base soon. Currently, we are using an in-house made tool [mgBench](https://github.com/memgraph/memgraph/tree/master/tests/mgbench). There are a lot of details there, feel free to ask for specifics.

**Comment by antejavor** [comment, marked as answer]
Running benchmarks on Memgraph will be recommended via [mgBench](https://github.com/memgraph/memgraph/tree/master/tests/mgbench), but you can use to tool you like the most. We are currently working on improving infrastructure and ease of use for running [benchmarks](https://github.com/orgs/memgraph/projects/4) by using mgBench.

**Comment by gitbuda** [comment]
A quick note from my side 😄

Our community has enthusiastically embraced [Discord](https://discord.gg/memgraph) and [Stackoverflow](https://stackoverflow.com/questions/tagged/memgraphdb) as the favored channels for sharing with our team and asking questions. We’ve decided to disable the discussions tab here on GitHub in 2 weeks.

We look forward to seeing you there!
